### DISCUSSION TITLE
Moes Star Ring ZigBee Smart Dimmer Switch

### DISCUSSION THREAD

**Discussion by micheledistefano79** [Koenkk/zigbee2mqtt#18609] [Devices]
Hello everyone,

is this device compatible with Zigbee2mqtt? Am I gonna be able to switch the status led off (during the night)?
https://moeshouse.com/products/moes-star-ring-zigbee-smart-dimmer-switch-for-light-dimming-work-with-alexa-google-home-dimmable-1-3gang

**Comment by demcas** [comment]
I just tried pairing this device and it says unsupported.  From what I understand it used the same internal sas this model https://www.zigbee2mqtt.io/devices/TS0601_dimmer_1.html

**Reply by micheledistefano79** [reply]
Thank you for trying and letting me know!

**Reply by demcas** [reply]
I'm new to this, how do we let the team know to raise this as a fix for a future release?
